workspace/utilities/site-variables.xsl

```diff
@@ -1,13 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:exslt="http://exslt.org/common"
+	exclude-result-prefixes="exslt">
 
-	<!-- Default site name with current language -->
-	<xsl:variable name="site-name">
-		<xsl:call-template name="default-site-name" />
-	</xsl:variable>
+<xsl:variable name="data" select="/data" />
```

Comment:
I do not see the point of having $data.

Reply:
We use a lot of exslt node sets and not having to care about having access to /data is way easier for everyone.